Automattic/woocommerce-payments
# Added order note when payment fails mentioning the failure reason.

Fixes #489 

#### Changes proposed in this Pull Request

* Adds order note when payment fails mentioning the failure reason.

#### Testing instructions

* Place an a order and use 4000000000009995 for payment.
* Ensure you have a note in order specifying why transaction has failed.

-------------------
